website/assets/scss/_styles_project.scss

```
@@ -1,3 +1,10 @@
 code {
   --bs-code-color: #{$secondary};
 }
+
+.td-navbar .navbar-brand svg {
+    display: inline;
+    margin: 0 10px;
+    height: calc(56px - 6px);
+    width: fit-content;
+}
```

```suggestion
.td-navbar .navbar-brand svg {
    width: fit-content;
}
```

Since CSS is a cascading language, I think we can leave the `display` / `margin` / `height` stuff off (since that's inherited from the base theme and we don't want to redefine it) and insert only the override we need -- the `width` property.  That should allow us to automagically grow in height (and whatever else) should our theme module ever update.

---

Updated to only add width style for the navbar logo.